docs/products/metrics/howto/download-influxdb-data.md

```diff
@@ -0,0 +1,162 @@
+
+## Prerequisites
+
+Before you download a backup, make sure you have the following:
+
+- [Aiven API token](/docs/platform/howto/create_authentication_token) with the necessary
+  permissions
+- [Aiven CLI](/docs/tools/cli)
+
+If your InfluxDB service is in a Virtual Private Cloud (VPC) or if you need specific
+network access to your backup, confirm that the correct access settings are enabled.
+Based on your network setup, enable one of the following settings:
+
+- `public_access.user_backup`: Allows access to backups from the public internet
+- `private_access.user_backup`: Allows access to backups within a private network
+- `privatelink_access.user_backup`: Allows access to backups via a private link
+
+To configure access:
+
+1. Go to the service's <ConsoleLabel name="overview" /> page.
+1. Click <ConsoleLabel name="service settings" />, then go to **Cloud and network**.
+1. Click the <ConsoleLabel name="actions" /> menu and select
+   **More network configurations**.
+1. Click <ConsoleIcon name="Add config options"/>.
+1. In the search field, enter the access setting to enable. For example,
+   `private_access.user_backup`.
+1. Select the value to **Enabled**.
+1. Click **Save configuration**.
+
+## Backup format
+
+The backup created is a portable file that includes all databases and retention policies
+from your Aiven for InfluxDB service. This backup is packaged as a single `.tar` file
+for easy storage and transfer. For more information, see the
+[InfluxDB backup and restore documentation](https://docs.influxdata.com/influxdb/v1/administration/backup_and_restore/).
+
+## Download data
+
+<Tabs groupId="method">
+<TabItem value="console" label="Aiven Console">
+
+1. Log in to the [Aiven Console](https://console.aiven.io/), select your project, and
+   choose your Aiven for InfluxDB service.
+1. On the service's <ConsoleLabel name="overview"/> page, click
+  <ConsoleLabel name="service settings"/>.
+1. In the  **Service management** section, click <ConsoleLabel name="actions"/> >
+   **Download data**.
+1. In the **Download InfluxDB data** window, review the details and enter a name
+   in the **Backup name** field. Use only letters, numbers, and the characters `.:_+-`
+   without spaces or parentheses.
+1. Click **Request backup** to create a backup of your latest data.
+
+   :::note
+   The duration of the backup process varies based on the size of your data. For larger
+   instances, it may take several hours. Requesting a new backup will replace any
+   existing backup, so download any existing backup you need before creating a new one.
+   :::
+
+1. After the backup is created, copy the **Service URI** to access the backup files
+   directly. Paste the URI into your browser to download your backup.
+
+   The **Service URI** provides direct access to the backup files. To access the backup
```

Comment:
I do not know how the current UI looks like now, but what "Service URI" this passage refers to? If it is InfluxDB Sevice URI, like in the streenshot, then it is not correct. Or is it some other Service URI?
![image](https://github.com/user-attachments/assets/e24890e9-c281-4657-bacd-8039b3934491)

Reply:
This is the screen users see when they want to download the data:
![image](https://github.com/user-attachments/assets/69045945-c584-49cf-87ee-1529da69ea68)

The Service URI referenced here pertains to this screen..

Reply:
**Request download**  button will be updated to **Request backup**.  Mostly in this [PR](https://github.com/aiven/aiven-core/pull/52801)

Reply:
@cincinnat, let me know if this clarifies your question.

Reply:
Yes, thanks.